akka-docs/src/test/java/jdocs/tutorial_3/Device.java

```diff
@@ -82,16 +80,6 @@ public void postStop() {
   @Override
   public Receive createReceive() {
     return receiveBuilder()
-            .match(RequestTrackDevice.class, r -> {
-              if (this.groupId.equals(r.groupId) && this.deviceId.equals(r.deviceId)) {
-                getSender().tell(new DeviceRegistered(), getSelf());
-              } else {
-                log.warning(
-                        "Ignoring TrackDevice request for {}-{}.This actor is responsible for {}-{}.",
-                        r.groupId, r.deviceId, this.groupId, this.deviceId
-                );
-              }
-            })
```

Comment:
I'm not sure that's needed: Ruth didn't make any changes to those files, and I just moved them without further changes